app/src/main/java/com/example/ar_reshare/Product.java

```diff
@@ -96,6 +88,14 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public int getContributorID() {
+        return contributorID;
+    }
+
+    public void setName(int contributorID) {
```

Comment:
could you rename this to setContributorID please?

Reply:
Good catch Hellin! I've replaced this now